scripts/india_nhm/districts/nhm_maternal_health/NHM_MaternalHealth.mcf

```diff
@@ -0,0 +1,56 @@
+Node: dcid:Count_PregnantWomen_RegisteredForAntenatalCare
+description: "Total number of pregnant women registered for Antenatal Care"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_PregnantWomen_RegisteredForAntenatalCare
+statType: dcs:measuredValue
+
+Node: dcid:Count_PregnantWomen_RegisteredForAntenatalCareWithinFirstTrimester
+description: "Number of pregnant women registered for Antenatal Care within first trimester"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_PregnantWomen_RegisteredForAntenatalCareWithinFirstTrimester
+statType: dcs:measuredValue
+
+Node: dcid:Count_ChildDeliveryEvent
+description: "Total reported child deliveries"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_ChildDeliveryEvent
+statType: dcs:measuredValue
+
+Node: dcid:Count_ChildDeliveryEvent_InAnInstitution
+description: "Institutional deliveries (includes public and private institutions)"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_ChildDeliveryEvent_InAnInstitution
+statType: dcs:measuredValue
+
+Node: dcid:Count_ChildDeliveryEvent_InPublicInstitution
+description: "Deliveries conducted at public institutions"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_ChildDeliveryEvent_InPublicInstitution
+statType: dcs:measuredValue
+
+Node: dcid:Count_ChildDeliveryEvent_AtHome
+description: "Number of home deliveries"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_ChildDeliveryEvent_AtHome
+statType: dcs:measuredValue
+
+Node: dcid:Count_ChildDeliveryEvent_AtHome_WithStandByAssist
+description: "Number of home deliveries attended by StandBy Assist (Doctor/Nurse/ANM)"
```

Comment:
nit: standby is one-word too, so
_WithStandbyAssist
Number of home deliveries with Standby Assistance (Doctor / Nurse / ANM)

also.. what is ANM

Reply:
From Wikipedia:

> Auxiliary Nursing Midwife (ANM) is a village-level female health worker in India who is known as the first contact person between the community and the health services.

We have common midwifes for villages who are not clinically authorized to deliver babies but they have experience in doing so. Most villages have one female villager supervising all home deliveries.

Reply:
Renamed to `Standby`
ANM is `Auxiliary Nurse Midwife`, also called ASHA a form of community health worker.
Expanded in the description